packages/zendesk-adapter/src/filters/reorder/trigger.ts

```diff
@@ -59,7 +59,11 @@ const deployFunc: DeployFuncType = async (change, client, apiDefinitions) => {
   const instance = getChangeData(clonedChange)
   const { order } = instance.value
   if (!areTriggerOrderEntries(order)) {
-    throw new Error('trigger_order\' order field has an invalid format')
+    throw createSaltoElementError({
+      message: 'trigger_order\' order field has an invalid format',
```

Comment:
not specific for this PR, but can we fix the typo please? :) 
```suggestion
      message: 'trigger_order\'s order field has an invalid format',
```